numpy/lib/tests/test_io.py

```diff
@@ -633,6 +633,15 @@ def test_usecols(self):
         x = np.loadtxt(c, dtype=float, usecols=np.array([1, 2]))
         assert_array_equal(x, a[:, 1:])
 
+        # Testing with an integer instead of a sequence
```

Comment:
If you want to test with something really crazy, could test with:
```
class CrazyInt(object):
     def __index__(self):
         return 0
```
but I am OK with leaving it out.